.github/workflows/push-tag-create-release.yml

```diff
@@ -39,10 +39,9 @@ jobs:
           cd build-tools
           pnpm install --frozen-lockfile
           pnpm run build:compile
-
-          # add the bin dir of build-cli to the path
-          cd packages/build-cli/bin
-          echo "$(pwd)" >> $GITHUB_PATH
+          cd packages/build-cli
+          # Use npm link instead of pnpm link because it handles bins better
+          npm link
```

Comment:
```suggestion
          # We want flub available to call, so we run npm link in the build-cli package, which creates shims that are avilable on the PATH
          # Use npm link instead of pnpm link because it handles bins better
          cd packages/build-cli
          npm link
```